api/kyverno/v1alpha2/cleanuppolicy_types.go

```diff
@@ -0,0 +1,32 @@
+package v1alpha2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Schedule",type=string,JSONPath=".spec.schedule"
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+
+// CleanupPolicy declares cleanup behaviors for matching resources.
+type CleanupPolicy struct {
+	metav1.TypeMeta   `json:",inline,omitempty"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec declares policy behaviors.
+	Spec CleanupSpec `json:"spec"`
+
+	// Status contains policy runtime data.
+	// +optional
```

Comment:
is this optional ?

Reply:
I think it should be since we will be updating the status during the reconciliation of the CR. WDYT?

Reply:
I think we can remove the optional marker. It’s a sub resource anyway.